examples/2-handle-prompt/index.js

```diff
@@ -0,0 +1,100 @@
+import http from 'http';
+import { Octokit } from "@octokit/rest";
```

Comment:
Do not use `@octokit/rest`, use `octokit` instead. I know Copilot likes to recommend `@octokit/rest` because it's been around longer, but it's a legacy SDK. `octokit` is the official all-batteries-included SDK for GitHub's API now